src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -8,7 +9,8 @@
     <AssemblyName>Microsoft.MIDebugPackage</AssemblyName>
     <OutputPath>$(MIDefaultOutputPath)</OutputPath>
 
-    <EnableDefaultEmbeddedResourceItems Condition="'$(IsXPlat)'=='true'">false</EnableDefaultEmbeddedResourceItems>
+    
+    <EnableDefaultEmbeddedResourceItems Condition="'$(MSBuildRuntimeType)' == 'Core'">false</EnableDefaultEmbeddedResourceItems>
```

Comment:
>Condition="'$(MSBuildRuntimeType)' == 'Core'" [](http://example.com/codeflow?start=40&length=45)

This project isn't in MIDebugEngine-Unix.sln, so is there any value in making changes to this project?

Reply:
Oh, I see. Nevermind.